portal-api/src/v1/gef/controllers/application-submit.js

```diff
@@ -195,7 +196,10 @@ const addSubmissionData = async (dealId, existingApplication) => {
   const updatedPortalActivity = await submissionPortalActivity(existingApplication);
   await addSubmissionDateToIssuedFacilities(dealId);
   await addUkefFacilityIdToFacilities(dealId);
-  await removeChangedToIssued(dealId);
+  // if MIA then handled by central API
```

Comment:
what do you mean by this, what's handled by central API?

Reply:
if MIA, central-api, does it from tfm-api and central-api